packages/react-charting/src/components/DonutChart/DonutChart.test.tsx

```diff
@@ -123,10 +123,39 @@ describe('DonutChart - mouse events', () => {
   beforeEach(sharedBeforeEach);
   afterEach(sharedAfterEach);
 
-  it('Should render callout on hover', () => {
+  it('Should render callout correctly on mouseover', () => {
     wrapper = mount(<DonutChart data={chartPoints} innerRadius={55} calloutProps={{ doNotLayer: true }} />);
     wrapper.find('path[id^="_Pie_"]').at(0).simulate('mouseover');
     const tree = toJson(wrapper, { mode: 'deep' });
     expect(tree).toMatchSnapshot();
   });
+
+  it('Should render callout correctly on mousemove', () => {
+    wrapper = mount(<DonutChart data={chartPoints} innerRadius={55} calloutProps={{ doNotLayer: true }} />);
+    wrapper.find('path[id^="_Pie_"]').at(0).simulate('mousemove');
+    const textContent1 = wrapper.find('.ms-Callout').hostNodes().text();
+    wrapper.find('path[id^="_Pie_"]').at(0).simulate('mouseleave');
+    wrapper.find('path[id^="_Pie_"]').at(1).simulate('mousemove');
+    const textContent2 = wrapper.find('.ms-Callout').hostNodes().text();
+    expect(textContent1).not.toBe(textContent2);
```

Comment:
Add a comment on what is the expected text

Reply:
Okay